### ISSUE TITLE
SASS (SCSS) @import never works when following official documentation

### ISSUE THREAD

**Issue by Fusty** [dotCMS/core#15001]
Any time an @import statement is used in an SCSS file dotCMS will fail to produce the compiled CSS.

## Expected Behavior
Per the [documentation](https://dotcms.com/docs/latest/server-side-compiling-of-sass-files#ImportingFiles) there are several ways to specify a file using the @import statement.  In this example we will have a master SCSS file called demo.scss and a partial file called _underscore.scss.  I'd assume all of the suggested methods would work, but it seems that none of them do and I also cannot find any way of tricking it into working.

## Current Behavior
1.  `@import "underscore"`  - is the first (and most idiomatic) suggestion.  This does not work.  It results in an error in dotcms.log stating that demo.scss cannot be found or read at .../assets/tmp_upload/css_compile_space/fe83cf2e-79b4-4902-873e-3435daea236d/css/demo.scss

2.  `@import "/css/_underscore.scss"` or `@import "/css/underscore"` - both behave the same, and result in the same error as above (note no mention of the @import statement in the error)

3.  `@import "//demo.dotcms.com/css/_underscore.scss"` or `@import "//demo.dotcms.com/css/underscore"` - Results in the same behavior as both of the above.

## Possible Solution
Fix whatever issue dotCMS has writing into the directories it expects to.  It complains about not being able to read files in a specific directory, and when I look for those files (on my local instance) they do not exist as well as the parent folder.  All of this manifests on demo.dotcms.com (except the on-disk presence, I cannot check this easily).  In my experiments locally file permissions, symbolic links and different filesystems don't seem to have any effect on the issue.

## Known Workarounds
NONE - This is completely crippling this feature and making it useless.  @import statements are arguably SASS' killer feature

## Steps to Reproduce (for bugs)
Create a file in /css/ called `_underscore.scss` and in it put the following
```
$color: black;
```

Create a file in /css/ called `demo.scss` and in it put the following
```
@import "underscore";

body {
	background: $color;
}
```
OR
```
@import "underscore.scss";

body {
	background: $color;
}
```
OR
```
@import "_underscore.scss";

body {
	background: $color;
}
```
OR
```
@import "/css/_underscore.scss";

body {
	background: $color;
}
```
OR
```
@import "/css/underscore";

body {
	background: $color;
}
```
OR
```
@import "//demo.dotcms.com/css/underscore";

body {
	background: $color;
}
```
OR
```
@import "//demo.dotcms.com/css/_underscore.scss";

body {
	background: $color;
}
```

Try any of those, none will work.  Obviously I'm trying a lot of things because I would just like ANYTHING to work.

To view the file navigate to https://demo.dotcms.com/DOTSASS/css/demo.css

## Context
The lack of this feature means I cannot use @import statements in my own SASS files.  It also means I cannot use dotCMS to compile things like other CSS frameworks that are distributed in SASS.  This is because everyone who seriously utilizes SASS' features will use the @import statement frequently.

## Your Environment
https://demo.dotcms.com

**Comment by wezell** [comment]
This is possibly a result of the demo site's hosting environment not having access permissions to the underlying disk space needed to compile the sass.

**Comment by wezell** [comment]
I can confirm that this works in soon (oh so soon) to be released 5.x version

**Comment by Fusty** [comment]
Is there an RC for 5.x we can try yet @wezell?

**Comment by wezell** [comment]
Not an official RC but I feel very comfortable pointing you to the latest nightly to try

https://dotcms.com/contentAsset/raw-data/4f75b6e3-4d71-4359-b56f-452bd6eb993e/targz/dotcms-2018-07-30_22-16.tar.gz

**Comment by Fusty** [comment]
Thanks @wezell

**Comment by wezell** [comment]
fixed in 5.0.0
